src/main/java/io/excaliburfrc/robot/RobotContainer.java

```diff
@@ -32,9 +34,16 @@ public RobotContainer() {
   private void configureButtonBindings() {}
 
   void manualButton() {
+    final int motorAxis = 1;
+    final int anglePiston = 2;
```

Comment:
Shouldn't this be in Constants?

Reply:
should it?
@Starlight220

Reply:
Button bindings go in RobotContainer, this includes their port definitions. Splitting the button/axis indexes to a separate file will just make it harder to quickly see where and how buttons are bound.